### ISSUE TITLE
Zero-knowledge extension

### ISSUE THREAD

**Issue by ai** [logux/logux#61]
A tool to encrypt all actions (not from block-list) to hide them from server:

```js
import { encryptActions } from "@logux/client"

…

encryptActions(client, password)
```

- [x] `encryptActions`
- [x] Clients Docs
- [x] Action cleaning
- [x] Server solution in docs

**Comment by ai** [comment]
Fixed d134ba0

**Comment by ai** [comment]
Implementation https://github.com/logux/client/commit/d134ba03cfd2b60f9d2d9379ef108067f6df785a
